src/pylic/config.py

```diff
@@ -0,0 +1,35 @@
+from collections.abc import MutableMapping
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Any
+
+import toml
+
+
+@dataclass
+class Config:
+    safe_licenses: list[str] = field(default_factory=list)
+    unsafe_packages: list[str] = field(default_factory=list)
```

Comment:
Maybe while you’re at it, you could deprecate this and introduce the new config you mentioned in #120 ?

But if you need to come up with a new strategy to support the SPDX format anyway later, maybe it’s not needed for now.

Reply:
thx for the review, much appreciated!

Oh I forgot about that one, yes let me rethink and apply it already if it still makes sense.

Before releasing v5 I would like to add spdx support and also add some tests, but I need to check how to this nicely with the new typer setup. I will merge this soon, to unblock a few follow-ups.